protovalidate/validator.py

```diff
@@ -84,8 +84,34 @@ def collect_violations(
             constraint.validate(ctx, message)
             if ctx.done:
                 break
+        for violation in ctx.violations.violations:
+            if violation.HasField("field"):
+                violation.field.elements.reverse()
+            if violation.HasField("rule"):
+                violation.rule.elements.reverse()
+            violation.field_path = Validator._field_path_string(violation.field)
         return ctx.violations
 
+    @classmethod
+    def _field_path_string(cls, path: validate_pb2.FieldPath) -> str:
```

Comment:
nit: should we move this to `protovalidate.internal` and import it instead? doesn't need to be a classmethod AFAICS

Reply:
Sure, SGTM. Moved to `internal`.